src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/Models/SemanticTokensEdit.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp.Models
+{
+    /// <summary>
+    /// Class representing an individual edit incrementally applied to a previous
+    /// semantic tokens response from the Document provider.
+    /// </summary>
+    [DataContract]
+    public class SemanticTokensEdit
```

Comment:
Do we need to have the Edit class available now or is that something we can add once we add support for the edit endpoint?

Reply:
Actually, I just noticed this was in the LanguageServerClient.Razor/HtmlCSharp library. Why do we need this at all? I wont leave any more comments but I'm curious about why we need the types in the `Models` namespace in our languageserver.client project at all